workflows/scripts/aggregate_cells.py

```diff
@@ -94,28 +82,30 @@ def aggregate(
             "Unable to find associated platemap name with given plate barcode"
         )
 
+    # loading in indentified platemap
```

Comment:
Not sure what you mean by `indentified` (identified?)

Reply:
Yes it was a type! fixed!